packages/notifier/test/test-publish-kit.js

```diff
@@ -293,9 +495,16 @@ test('subscribeLatest', async t => {
     [1, 2, 4, 8, 16, 32],
     'only the last of values published between iteration steps should be observed',
   );
-});
+};
+
+for (const [type, maker] of Object.entries(makers)) {
+  test(`subscribeLatest(${type} subscriber)`, verifySubscribeLatest, maker);
+}
 
-test('subscribeEach', async t => {
+// TODO: Replace with test.macro once that works with prepare-test-env-ava.
```

Comment:
try using `prepare-test-env` instead of `prepare-test-env-ava` and `import test from 'ava';`

Reply:
No, it doesn't do anything to Ava.
https://github.com/Agoric/agoric-sdk/blob/master/packages/SwingSet/tools/prepare-test-env.js

Reply:
Success! Thanks!